duouniversal/client.go

```diff
@@ -91,7 +91,7 @@ func (f *FlagStatus) UnmarshalJSON(data []byte) error {
 }
 
 type HealthCheckTime struct {
-	Time int `json:"time"`
+	Time int `json:"timestamp"`
```

Comment:
Golang neophyte question; is there any concern about backward compatibility here?  or is this something that's been broken all along and so nobody is using/relying on?